pages/front.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { compose } from 'react-apollo'
+import Front from '../components/Front'
+import withData from '../lib/apollo/withData'
+import { enforceMembership } from '../components/Auth/withMembership'
+
+const FrontPage = ({ url, headers }) => {
+  return <Front url={url} headers={headers} />
```

Comment:
Path is hard-coded in [`<Front>`](https://github.com/orbiting/republik-frontend/blob/d78b56a67612e695e03d6375e475f91e3ee7387e/components/Front/index.js#L108). You'll need to make that dynamic, use [`url.asPath.split('?')[0]`](https://github.com/orbiting/republik-frontend/blob/d78b56a67612e695e03d6375e475f91e3ee7387e/components/Article/Page.js#L430) and please test carefully.